test-app/tests/integration/components/combobox-test.gts

```diff
@@ -0,0 +1,854 @@
+import { click, fillIn, render, triggerKeyEvent } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+
+import ComboboxControl from '@crowdstrike/ember-toucan-core/components/form/controls/combobox';
+import { setupRenderingTest } from 'test-app/tests/helpers';
+
+let testColors = ['blue', 'red'];
+
+module('Integration | Component | Combobox', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(<template><ComboboxControl data-combobox /></template>);
+
+    assert.dom('[data-combobox]').hasTagName('input');
+    assert.dom('[data-combobox]').hasClass('text-titles-and-attributes');
+    assert.dom('[data-combobox]').hasClass('shadow-focusable-outline');
+    assert.dom('[data-combobox]').doesNotHaveClass('text-disabled');
+    assert.dom('[data-combobox]').doesNotHaveClass('shadow-error-outline');
+    assert
+      .dom('[data-combobox]')
+      .doesNotHaveClass('focus:shadow-error-focus-outline');
+
+    assert.dom('[data-combobox]').hasAttribute('aria-autocomplete', 'list');
+    assert.dom('[data-combobox]').hasAttribute('aria-haspopup', 'listbox');
+    assert.dom('[data-combobox]').hasAttribute('autocapitalize', 'none');
+    assert.dom('[data-combobox]').hasAttribute('autocomplete', 'off');
```

Comment:
Good question!

It is a little confusing, especially since we are eventually wanting to rename this to "autocomplete" instead.  We want to turn this attribute `off`, as this is the browser's default interactions as defined at https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete.

> The browser is not permitted to automatically enter or select a value for this field. It is possible that the document or application provides its own autocomplete feature, or that security concerns require that the field's value not be automatically entered.

That last part is the important piece - we are defining our own autocomplete feature via this component. By us setting `autocomplete="off"` here, we are telling the browser **not** to suggest autocomplete options.  Rather, _we_ provide the suggestions manually via the popover contents.  

Maybe as a TL;DR: we want to disable the browser's built-in autocomplete and use our own implementation instead.